src/components/SignIn/SignInPage.js

```diff
@@ -1,23 +1,37 @@
 import './SignInPage.css'
-import {Link} from 'react-router-dom'
+import {Link, Redirect} from 'react-router-dom'
 import { Component } from 'react';
 
+import { userSignIn } from '../../APICalls.js'
+
 class SignInPage extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      email: '',
+      username: '',
       password: '',
+      redirect: false
```

Comment:
Very cool! I like how you addressed the `redirect` functionality for the sign in.!